package.json

```diff
@@ -30,6 +30,7 @@
     "@ember/test-helpers": "^2.6.0",
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
+    "@lblod/ember-environment-banner": "github:lblod/ember-environment-banner#0.1.0",
```

Comment:
I would try to get the npm release out before merging this.

Reply:
Yes that would be ideal, I will convert this PR to a draft until the package has been released.